examples/problems/800_score_genes_for_marginals.ipynb

```diff
@@ -209,6 +209,8 @@
     "\n",
```

Comment:
<p>please include the information about the time. you can check out the corresponding notebook of WOT: <a href="https://broadinstitute.github.io/wot/tutorial" target="_blank">https://broadinstitute.github.io/wot/tutorial/</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>